ignite/pkg/cosmosanalysis/app/parse.go

```diff
@@ -3,178 +3,224 @@ package app
 import (
 	"go/ast"
 	"go/build"
-	"go/parser"
-	"go/token"
-	"os"
-	"path/filepath"
-	"strings"
-)
-
-func parseImports(n []*ast.ImportSpec) map[string]string {
```

Comment:
The logic was already present in `goanalysis.FindImportedPackages`, so I created `goanalysis.FormatImports` to replace that function.